std/math.d

```diff
@@ -7616,6 +7616,71 @@ if (isFloatingPoint!(F) && isIntegral!(G))
     }
 }
 
+@safe @nogc nothrow pure unittest
+{
+    float f1 = 19100.0f;
+    float f2 = 0.000012f;
```

Comment:
This could still be evaluated at compile time so best make it static or __gshared.

Reply:
OK, let's see, if that works...

Reply:
It wasn't a suggestion to make something work, it was a suggestion to defeat gdc/ldc's ability to constfold the entire test away. :-)

Reply:
Yeah, of course. But it might also have been the reason for the breaking. (But wasn't obviously.)